stable/airflow/UPGRADE.md

```diff
@@ -1,5 +1,12 @@
 # Upgrading Steps
 
+## `v7.2.0` → `v7.3.0`
+
+__The following values have been ADDED:__
+
+* `flower.service.nodePort`
```

Comment:
I would say this `flower.service.nodePort` is not really "a value that has been added"

Reply:
changed it to an improvement

Reply:
Actually, I just understood what you meant, it is just the section the `http` value is under. I took off that line now.